.github/workflows/integration.yml

```diff
@@ -8,14 +8,12 @@ jobs:
         os: [ "ubuntu" ]
         go: [ "1.24.x", "1.25.x" ]
         race: [ false ]
-        use32bit: [ false ]
         include:
           - os: "ubuntu"
             go: "1.25.x"
             race: true
           - os: "ubuntu"
             go: "1.25.x"
-            use32bit: true
```

Comment:
I think this entire configuration can be deleted, otherwise we'll be testing on Ubuntu twice right?